src/netex_idf/stops.rs

```diff
@@ -271,27 +108,20 @@ fn load_stops(
         .flat_map(|e| e.children())
         .collect();
```

Comment:
I can't pass an iterator in load_stop_areas, I already tried but there is an error. "value moved" stuff

Reply:
I was thinking about something like this.

Reply:
done

Reply:
Well almost... The PR I created was sending just a reference of the `Vec`. You don't need to send the entire `Vec` to the function.

Reply:
for now load_stop_area can take the ownership. I don't use stop_areas elsewhere.
I agree I will later I will need a reference. this will be done in another PR